### ISSUE TITLE
What exactly is a workshop?

### ISSUE THREAD

**Issue by FaceDeer** [Thomas--S/ts_workshop#1]
The readme file for this mod doesn't actually explain what the mod does. I had a skim through the code and it appears to be about setting up nodes that auto-creates items in some way?

**Comment by Thomas--S** [comment]
Yes, this mod provides an API for nodes that provide an alternative crafting method.

You can find examples for workshops in the [ts_doors](/minetest-mods/ts_doors) and the [streets](/minetest-streets/streets) mods.

If there are further questions, feel free to ask.

I know that this mod is not well documented, but I didn't find time for writing a good documentation yet. This mod was mainly written for the internal usage in the two mods mentioned above.

**Comment by FaceDeer** [comment]
Alright, well, I guess this issue will be the "needs documentation" issue, then. :) I've been working on my own alternative crafting system and workshops mods so when I saw this I was keen to investigate, but the readme file was unilluminating.
